### ISSUE TITLE
Layout/EmptyLinesAroundArguments reported on a line without arguments

### ISSUE THREAD

**Issue by agross** [bbatsov/rubocop#5274]
## Expected behavior

No issues.

## Actual behavior

Message: `Layout/EmptyLinesAroundArguments: Empty line detected around arguments.`

## Steps to reproduce the problem

I use this module to work around some issues with Bundler and Cygwin git.

```rb
# frozen_string_literal: true

module Bundler
^
  module Cygwin
    module GitSupport
      def git_retry(command)
        return super(command) unless Gem.win_platform? && system('which cygpath >NUL 2>NUL')

        command = command.split(' ').map do |maybe_path|
          next maybe_path unless maybe_path =~ %r{^"[A-Z]:/}i

          `cygpath --unix #{maybe_path}`.strip
        end.join(' ')

        super(command)
      end
    end
  end
end
```

## RuboCop version

```
$ rubocop -V
0.52.0 (using Parser 2.4.0.2, running on ruby 2.4.1 x64-mingw32)
```

**Comment by benhutton** [comment]
I'm getting really weird results here too. Lots of different files breaking in different types of places. Would be happy to give some more examples if needed.

**Comment by garettarrowood** [comment]
This has been resolved but is not yet in a released version.Resolved in #5231 .
